NEWS

```diff
@@ -26,6 +26,10 @@ Several important changes and enancements were added:
      cat /proc/self/maps | grep '\[vvar]'
 * Support for glibc version 2.21 added.  To see if this affects you, do:
      ls -l /lib*/libc.so.6 /lib/*/libc.so.6
+* The environment variable DMTCP_GDB_ATTACH_ON_RESTART was added.  Setting
```

Comment:
Hmm. I am not sure if we can ever support `gdb --args dmtcp_restart ckpt*.dmtcp` to work normally. However, I agree that the user should be able to "attach" to their processes after restart. In that case, wouldn't it be better to also supply a command line flag to dmtcp_restart to complement this env var? I personally would prefer to use a flag over an environment variable :-). Having said that, let's commit it and create a new issue to add the command line flag.